CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
 # Change Log (Release Notes)
 
 All _notable_ changes to this project will be documented in this file (`CHANGELOG.md`).
-This project adheres to [the Python form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme) (or at least we try).
+
+This project adheres to [Semantic Versioning 2](http://semver.org/spec/v2.0.0.html), the BigchainDB **public API**:
```

Comment:
There are some small differences between [Semantic Versioning 2](http://semver.org/spec/v2.0.0.html) and [Python's form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme). The main difference is in the version label string (point 9 in Semantic Versioning). This matters to pip and PyPI.

Note: I thought I made this comment already yesterday but for some reason it didn't get saved.